src/api/comms.js

```diff
@@ -72,6 +80,48 @@ export function put(uri, body) {
   });
 }
 
+export function patchAddSecretBody(secretName) {
```

Comment:
Method name's a little confusing, is this generateBodyForSecretPatching?

Reply:
We've updated this to the name you suggested